### PR TITLE
OMRSJS-18 : Fixed Unknown Provider Error with "angular-animate"

### PR DESCRIPTION
**Pull request by Ankitkumar94**
Fixed the "Unknown Provider Error" by adding angular-animate : "^1.6.4" dependency in the package.json (of both generator and template) file and also made the angular version to be inline with the angular-animate version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
     "owa"
   ],
   "dependencies": {
+    "angular": "^1.6.4",
+    "angular-animate": "^1.6.4",
```
Comment (psbrandt) [inline package.json]: What's the reason for adding this? The generator itself just scaffolds the app based on the templates and user input, it doesn't use angular or any other frontend framework.

Comment (Ankitkumar94) [inline package.json]: Its by mistake. I ran the command "npm install --save" in the main directory and forgot to remove !!
Let me send another pull request !

Comment (psbrandt) [inline package.json]: Okay. A new PR is fine for now, but just pushing a commit to Ankitkumar94:OMRSJS-18 would have updated the PR.
